Library/Homebrew/extend/ENV/shared.rb

```diff
@@ -171,7 +171,11 @@ def compiler
 
   # @private
   def determine_cc
-    COMPILER_SYMBOL_MAP.invert.fetch(compiler, compiler)
+    if compiler == :brew_clang
+      "/usr/local/opt/llvm/bin/clang"
```

Comment:
Yeah. I agree that the approach is lacking.

Is `HOMEBREW_PREFIX` preferred over `brew --prefix`?